src/sql/parts/modelComponents/toolbarContainer.component.ts

```diff
@@ -85,6 +88,10 @@ export default class ToolbarContainer extends ContainerBase<ToolbarItemConfig> i
 		return this.hasTitle(item) && this.isHorizontal();
 	}
 
+	public shouldShowToolbarSeparator(item: ToolbarItem): boolean {
+		return item.config.toolbarSeparatorAfter;
```

Comment:
Check config isn't undefined

Reply:
can be simplified to: return item && item.config ? item.config.toolbarSeparatorAfte r: false;